contracts/src/v0.8/llo-feeds/v0.4.0/Configurator.sol

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Can we move all the new contracts from v0.4.0 to v.0.5.0 directory overwriting the existing contracts will cause dependency issues and make us lose track of which versions we have.